01_P/P_3_1_1_01/sketch.js

```diff
@@ -47,13 +48,11 @@ function setup() {
 
   font = "Arial";
 
-  smooth();
```

Comment:
Removed `smooth()` since this is already enabled by default in every p5 sketch (as well as Processing 2 and 3).